src/Illuminate/Support/Arr.php

```diff
@@ -120,7 +120,7 @@ public static function exists($array, $key)
             return $array->offsetExists($key);
         }
 
-        return array_key_exists($key, $array);
+        return isset($array[$key]) || array_key_exists($key, $array);
```

Comment:
…and even if it's slower, the difference is [super micro](https://ilia.ws/archives/247-Performance-Analysis-of-isset-vs-array_key_exists.html) and just not worth it if I may add. Unless, of course, you're calling this method repeatedly for like 10,000 times.